config/routes.rb

```diff
@@ -34,4 +34,6 @@
   get '/applications/new', to: 'applications#new'
   post '/applications', to: 'applications#create'
   get '/applications/:id', to: 'applications#show'
+
+  patch '/applications/:application_id/pets/:pet_id', to: 'application_pets#update'
```

Comment:
my reasoning for this routing is that approving an application for a single pet is basically updating that record in `ApplicationPet` - not the application itself. adjusted the URI to include both ids so an ApplicationPet lookup can be made in the controller